### PR TITLE
Feat/issue notification/add tooltip feature in issue notification

### PR DESCRIPTION
**Pull request by Sandeep1908**
**Pull Request Requirements:** add a tooltip feature in IssueNotification

Screenshot 2024-04-08 at 2 41 14 AM

* Provide a brief description of the changeset.
* Include a screenshots if applicable
* Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjEzMGM1YTY4ZWY5NjM0ZDNmMDQ0NTEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.VL9v2RmM-DBl88DpUHO68jtnMWHS_uBywtq1ebVobgE">Huly&reg;: <b>UBERF-6404</b></a></sub>

### CODE REVIEW

File: plugins/tracker-resources/src/components/issues/IssueNotification.svelte [hunk 1]
```diff
@@ -71,8 +82,26 @@
     </div>
   </svelte:fragment>
 
-  <svelte:fragment slot="buttons">
+  <svelte:fragment slot="buttons"  >
     <Button label={tracker.string.ViewIssue} on:click={handleIssueOpened} />
     <Button icon={view.icon.CopyLink} label={tracker.string.CopyIssueUrl} on:click={handleCopyUrl} />
+
+    {#if showCopiedTooltip}
+      <div class="tooltip">Copied URL!</div>
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/IssueNotification.svelte:90]: String should be internationalised.

File: rush.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
      * The default value is false to avoid legacy compatibility issues.
      * It is strongly recommended to set strictPeerDependencies=true.
      */
-    "strictPeerDependencies": false,
```
Comment (haiodo) [inline rush.json:65]: Why changes to this file are done?

File: plugins/tracker-resources/src/components/issues/IssueNotification.svelte (hunk 1 above)
Comment (Sandeep1908) [reply plugins/tracker-resources/src/components/issues/IssueNotification.svelte:90]: Hi, I am not able to get the value internationalised, could you please help me out in this?

Comment (Sandeep1908) [reply plugins/tracker-resources/src/components/issues/IssueNotification.svelte:90]: @haiodo

**Comment by aonnikov** [comment]
Closed due to inactivity

**Pull request closed without merging**